dts/bindings/dma/nordic,easydma-device.yaml

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+# Common fields for all nRF peripherals featuring EasyDMA
+
+properties:
```

Comment:
Rather than putting this here, can you mimic what is done in linux and add `bindings/reserved-memory/memory-region.yaml `

See: https://git.kernel.org/pub/scm/linux/kernel/git/devicetree/devicetree-rebasing.git/tree/Bindings/reserved-memory/memory-region.yaml

Reply:
moved `dts/bindings/dma/nordic,easydma-device.yaml` -> `dts/bindings/reserved-memory/memory-region.yaml`